docker/Dockerfile.scancode

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+FROM python:3.6.12-buster
```

Comment:
Suggest using debian:buster as it's easier to get sources for the packages so we can ship convenience images in the future.

Reply:
Installing python on debian:buster is not trivial. But I agree, using `debian:buster` is better. I will rewrite this.

Reply:
I'm struggling to get python installed on buster.. sorry for the delay!

Reply:
Why not install python3 using apt?

Reply:
Scancode requires python3.6 and that does not seem to be available anymore.